lucet-module/src/error.rs

```diff
@@ -31,4 +32,6 @@ pub enum Error {
     UnknownModule { module: String, symbol: String },
     #[error("Unknown symbol `{module}::{symbol}`")]
     UnknownSymbol { module: String, symbol: String },
+    #[error("Unknown bindings {unknown_bindings}")]
+    UnknownBindings { unknown_bindings: UnknownBindings },
```

Comment:
If you add a `#[from]` here, decls.rs:201 could be a bit simpler

Reply:
Done, thanks!